web/modules/custom/joinup_core/tests/src/ExistingSite/FileUrlFieldTest.php

```diff
@@ -203,10 +212,6 @@ protected function drupalLogin(AccountInterface $account) {
       'pass' => $account->passRaw,
     ], $this->t('Sign in'));
 
-    // @see BrowserTestBase::drupalUserIsLoggedIn()
-    $account->sessionId = $this->getSession()->getCookie($this->getSessionName());
-    $this->assertTrue($this->drupalUserIsLoggedIn($account), new FormattableMarkup('User %name successfully logged in.', ['%name' => $account->getAccountName()]));
```

Comment:
We're losing this but I guess it's not so important because we have extensive coverage for the log in procedure in the Behat scenarios. What was the reason this needed to be deleted?